src/frontend/next/src/pages/_document.tsx

```diff
@@ -12,10 +14,30 @@ class MyDocument extends Document {
   }
 
   render() {
+    const { title, description, author, telescopeUrl, keywords } = config;
     return (
       <Html lang="en">
         <Head>
           <link rel="icon" href={logoUrl} type="image/svg+xml" />
+          <meta charSet="utf-8" />
+          <meta name="theme-color" content={lightTheme.palette.primary.main} />
+
+          <meta name="description" content={description} />
+          <meta name="author" content={author} />
+          <meta name="keywords" content={keywords} />
+          <meta name="application-name" content={title} />
+
+          <meta property="og:type" content="website" />
+          <meta property="og:title" content={title} />
+          <meta property="og:site_name" content={title} />
+          <meta property="og:description" content={description} />
+
+          <meta name="twitter:title" content={title} />
+          <meta name="twitter:description" content={description} />
+          <meta name="twitter:image" content={logoUrl} />
+          <meta name="twitter:card" content="summary_large_image" />
+          <meta name="twitter:image:alt" content={title} />
```

Comment:
Great work @tonyvugithub! Here is one thing I've noticed. I think we might not need to provide an alt to the logo. I've checked [WCAG standards](https://www.w3.org/WAI/tutorials/images/) and it seems that this one would fall under `Decorative` category. We should be careful with assigning alternates to images.